service_container/injection_types.rst

```diff
@@ -248,3 +248,121 @@ to setter injection but with these additional important problems:
 But, it is useful to know that this can be done with the service container,
 especially if you are working with code that is out of your control, such
 as in a third party library, which uses public properties for its dependencies.
+
+Static Injection
```

Comment:
I'm not sure about the term - I don't use that personally, nor am I sure that it's clear.
Any better proposal from anyone?

Reply:
Whiter injection? 

Not sure about this one too :/